server/tso/tso.go

```diff
@@ -191,6 +234,7 @@ func (t *TimestampOracle) UpdateTimestamp() error {
 	if typeutil.SubTimeByWallClock(t.lastSavedTime, next) <= updateTimestampGuard {
 		save := next.Add(t.saveInterval)
 		if err := t.saveTimestamp(save); err != nil {
+			tsoCounter.WithLabelValues("err_save_ts").Inc()
```

Comment:
ditto, should be different from line 134?

Reply:
they are the same reason.